### DISCUSSION TITLE
2021-10-20 Weekly Meeting 26 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#108] [Weekly Meetings]
Weekly Meeting 26 is happening on Wednesday October 20, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - https://github.com/omigroup/omigroup/issues/72
   - OMI Website
     - Website development resuming next Tuesday 1PM Eastern 10AM Pacific
   - glTF subgroup
   - Event proposals for the Fall/Early Winter
   - Follow-up on requests for participation:
     - IPSME RFP https://github.com/omigroup/omigroup/issues/107
     - [OMI Group Project Ideas](https://github.com/omigroup/omigroup/discussions/105)
     - OMI VKet Booth
       - Tuesday 5:30pm PT 
     - Metaverse Discovery and Mapping
       - OMI Hackathon?

Add ons: 
   - ... as you see other things of interest in discord, a convo, or a demo -- please capture a screenshot/context and drop into comments below.

**Comment by mrmetaverse** [comment]
Suggesting depriorizing the website check in until we have more to report. At the present moment omigroup.org is functional, and folks are split across other projects to get this moving forward. A backlog is established and a working group is formed to manage inbound requests, but there isn't much to report for the 3rd week in a row. 

Also Metaversity request for participation proposal will be submitted before next Wednesday. The initial group is still working out some details like why you should care and how you can get involved.

**Comment by robertlong** [comment]
Notes: 
Rather than host our own hackathon we may join the WebXR hackathon and sponsor a prize.
